packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1726,24 +1726,31 @@ Authenticators can be broadly classified into three kinds of factors. A factor r
+| `inactivityPeriod`   | String (ISO 8601) | The period of inactivity after which the user must be reauthenticated.                                                | N/A                                                                                               |
 
+#### Constraints
 Each nested constraint object is treated as a list, all of which must be satisfied. The top-level array is treated as a set, one of which must be satisfied.
 
 This can be read logically as: `( (1A && 1B) || (2A && 2B) )`
 
 The number of authenticator class constraints in each constraint object be less than or equal to the value of `factorMode`. If the value is less, there are no constraints on any additional factors.
 
+| Property            | Data Type              | Description                                                                                                             | Supported Values                                                                                  |
+| -------------------- | ----------------- | ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| `types`              | Array             | The Authenticator Types which are permitted.                                                                            | `SECURITY_KEY`, `PHONE`, `EMAIL`, `PASSWORD`, `SECURITY_QUESTION`, `APP`, `FEDERATED`                          |
+| `methods`            | Array             | The Authenticator Methods which are permitted.                                                                          | `PASSWORD`, `SECURITY_QUESTION`, `SMS`, `VOICE`, `EMAIL`, `FIDO2`, `PUSH`, `SIGNED_NONCE`, `OTP`, `WEBAUTHN`, `DUO`, `IDP` |
+| `hardwareProtection` | String            | Indicates whether any secrets or private keys used during authentication must be hardware protected and not exportable. This property can be set in `POSSESSION` constraint only.| `REQUIRED`, `OPTIONAL`                                                                            |
+| `deviceBound` | String            | Indicates whether device bound factors are required. This property can be set in `POSSESSION` constraint only. | `REQUIRED`, `OPTIONAL`                                                                            |
+| `phishingResistant` | String            | Indicates whether phishing resistant factors are required. This property can be set in `POSSESSION` constraint only. | `REQUIRED`, `OPTIONAL`                                                                            |
+| `userPresence` | String            | Indicates if user presence is required or not.  This property can be set in `POSSESSION` constraint only.| `REQUIRED`, `OPTIONAL`                                                                            |
```

Comment:
Did the descriptions of these properties come from someone on the dev docs team?  I wonder if we should try to better explain what they mean.

For example, the UI option for `userPresence` says something like "Whether the user is required to accept a consent prompt" or something like that.